docs/handbook/posthog-array.md

```diff
@@ -0,0 +1,51 @@
+# The PostHog Array 
+
+The PostHog Array is our release newsletter and blog post, it serves 2 purposes
+- Updating the community that a new version has been released and what new features and functionaloty they can expect if they update.
+- Updating the community on PostHog itself, what we have been doing, finding interesting as well as selecting favorite contributions from the community.
+
+This section of the Handbook will guide you through generate when PostHog is releasing a new version.
+
+## 1. What has happened since the last release?
+
+As part of releasing a new version you will need to figure out what has been [updated in the new release.](https://handbook.posthog.com/#/dev/development-process/release-new-version)
+
+- `git log --pretty=format:%s [old-version]..head` will give you the pull requests that are new, you will need to split them into 4 categories:
+
+- PostHog Updates
+	- Updates that fit this category are front-end changes that introduce new or updated functionality and new libraries or integrations (including community ones).
+- Performance Updates
+	- Updates that fit this category will be back-end changes that have improved performance (this could be speed or logic, performance is somewhat subjective) and select bug fixes that can be considered impactful to PostHog UX.
+- Changelog updates
+	- Updates that fit this category are bug fixes not included in the above or minor performance updates.
+- Non Changelog updates
+	- Updates that serve no impact on PostHog, examples include the PR to update the Changelog, or bump version prs
+
+## 2. Write the PostHog Array
+
+Create a new post in the WP PostHog domain, the title is always The PostHog Array followed by the version number. Categrorize this as the Weekly Update. The featured image should be a high quality image of a Hedgehog with a width of 2000 pixels.
```

Comment:
I'd add that the image needs to be available for commercial use without attribution, unsplash is a good place to use